CONTRIBUTING.md

```diff
@@ -1,36 +1,64 @@
-See this here post on raising a github issue:
-https://community.openfoodnetwork.org/t/how-to-raise-a-github-issue/912
-
 # Contributing
+We love pull requests from everyone. Any contribution is valuable, but there are two issue streams that we especially love people to work on:
+
+1) Our delivery backlog, is managed via a ZenHub board (ZenHub extensions are available for most major browsers). We use a Kanban-style approach, whereby devs pick issues from the top of the backlog which has been organised according to current priorities. If you have some time and are interested in working on some issues from the backlog, please make yourself known on the [#dev](https://openfoodnetwork.slack.com/messages/C2GQ45KNU) channel on Slack and we can direct you to the most appropriate issue to pick up.
+
+2) Our list of bugs and other self-contained issues that we consider to be a good starting point for new contributors, or devs who aren’t able to commit to seeing a whole feature through. These issues are marked with the `# good first issue` label.
+
+## Set up
+
+Set up your local development environment by following the appropriate guide from the `Development environment setup` section in the [developer wiki](https://github.com/openfoodfoundation/openfoodnetwork/wiki).
+
+Fork the repo using the `Fork` button in the top-right corner of this screen. Then add the your fork as a remote on your local machine:
+
+    cd ~/location-of-your-local-ofn-repo
+    git remote add your-username https://github.com/your-username/openfoodnetwork
 
-We love pull requests from everyone. Here are some instructions for
-contributing code to Open Food Network. See the [developer wiki](https://github.com/openfoodfoundation/openfoodnetwork/wiki) for more information.
+Fetch the latest version of `master` from the main repo:
 
-Fork, then clone the repo:
+    git fetch origin master
 
-    git clone git@github.com:your-username/openfoodnetwork.git
+Create a new branch on your local machine for (based on `origin/master`):
 
-Follow the instructions in README.markdown to set up your machine.
+    git checkout -b branch-name-here --no-track origin/master
 
-Make sure the tests pass:
+If you want to run the whole test suite, we recommend using a free CI service to run your tests in parallel. Running the whole suite locally in series is likely to take > 40 minutes. [TravisCI][travis] and [SemaphoreCI][semaphore] both work great in our experience. Either way, make sure the tests pass on your new branch:
 
     rspec spec
 
-Make your change. Add tests for your change. Make the tests pass:
+## Making a change
+
+Make your changes to the codebase. We recommend using TDD. Make changes and get the test suite back to green.
```

Comment:
what about `... Add a test, make changes and get the test suite back to green.` It's good for beginners to be explicit about the fact that the test goes first.